doc/config_options.txt

```diff
@@ -40,6 +40,12 @@ The number of seconds to wait after the instance started before starting the nex
 The instance with the highest value is started first.
 ```
 
+```{config:option} boot.debug_edk2 instance-boot
+:shortdesc: "Enable debug version of the `edk2`"
+:type: "string"
+The instance should use a debug version of the `edk2` (`boolean`).
```

Comment:
@mihalicyn Shall we add something to the docs on how to access logs from this debug build?

cc @ru-fu

Reply:
Yes, we should definitely add where to find the log file. If I understand the description of the PR correctly, we might also need to point out how the firmware must be called (or can we assume it's always using the correct name)?

Reply:
> Yes, we should definitely add where to find the log file. If I understand the description of the PR correctly, we might also need to point out how the firmware must be called (or can we assume it's always using the correct name)?

I don't think that we need to specify the firmware name in the LXD docs, because our docs assume that LXD is shipped as LXD snap, and firmware in the snap is always named in accordance with the LXD source code.

>@mihalicyn Shall we add something to the docs on how to access logs from this debug build?

I'm not sure, but it's up to you guys. Because this is a very-very specific thing to debug edk2, it's not something that normal user will do. My understanding is that this feature will be used on the advice from our side with a clear step-by-step instruction for a user about how to turn it on and where to take the logs and so on.

Reply:
I think we should have the path where the logs will be stored in the docs so that if you are away/not present then others can know this feature exists.

Reply:
Have done.